doc/user/content/responsible-disclosure-policy.md

```diff
@@ -0,0 +1,69 @@
+---
+title: "Responsible Disclosure Policy"
+description: "Materialize's responsible disclosure policy."
+menu: "main"
+weight: 600
+---
+
+It's extremely important to us at Materialize to ensure that user information is safe and secure. We welcome the contributions of external security researchers towards this end, and have developed this policy to uphold our legal responsibility to good-faith security researchers who provide us with their expertise.
+
+## Scope
+
+Materialize is on a biweekly release schedule and expects all users to upgrade to the latest available version within two weeks of a release. Therefore, Materialize's Vulnerability Disclosure Policy covers only issues affecting the latest [version and the version prior to it](../versions).
+
+This restriction applies to both the installed executable (`materialized`) and Materialize Cloud.
+
+## Out of scope
+
+The follow types of attack are out of scope:
+
+- Attacks on the Materialize website ([https://www.materialize.com](https://www.materialize.com)), excepting the Cloud console
+- Password, email, and account policies, such as email verification, reset link expiration, password complexity.
+- Attacks on physical security
+- Attacks that use social engineering, distributed denial of service, spam or applications of third parties.
```

Comment:
What are the third-party applications? Might be useful to clarify that a bit. We use a lot of third party software in the applications we provide and those are presumably in scope.